### DISCUSSION TITLE
Change Stack order of Snapped Canvas Items

### DISCUSSION THREAD

**Discussion by BuzzfeedBlue** [runelite/runelite#15635] [Ideas]
### Type

Other

### Operating System

Windows

### OS Version

windows 11

### Bug description

Sometimes, specific activities have a lot of overlays that you want to organize in a specific way with the snap points around the screen, but once snapped they will always be in the same order.

I would love to, for example, be able to have status boxes on top of other infoboxes but currently it always defaults to the bottom and cannot be changed.

If we could add a new menu option, key combo, or even first item is always on the bottom, next added is above and so on, it would be a nice qol improvement for screen management

**Comment by afrandari** [comment]
This, please! Anytime I use the snap boxes, lets say for the one top right of the chat box, they always get ordered opposite of what I want. 
Like in the image I provide below, I'd want The biggest to smallest overlay going up so basically reverse of their current order. Just anyway to custom order those would be amazing. <3
![Capture](https://github.com/runelite/runelite/assets/25909738/f203aba1-0259-48be-a2de-5949e580f81d)

**Comment by wwboynton** [comment]
I'm still hoping that this will get some attention, this would make the UI really navigable when using a larger number of overlay elements.
